src/tasks/AotCompilerTask/MonoAOTCompiler.csproj

```diff
@@ -22,4 +22,6 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+
+  <Target Name="GetFilesToPackage" />
```

Comment:
Why is this needed? Comment would be useful

Reply:
I think its required by the `ExpandProjectReferences` target in `Packaging.targets`, following the `Pack` target in `Sdk.Targets`. 

I had followed the convention that generated the `Microsoft.NETCore.BrowserDebugHost.Transport` NuPkg when creating the `Microsoft.NET.Runtime.Android.Sample.Mono`, `Microsoft.NET.Runtime.iOS.Sample.Mono`, and `Microsoft.NET.Runtime.wasm.Sample.Mono` NuPkgs, because otherwise `/Users/mdhwang/runtime_wasm_clean/src/tasks/AotCompilerTask/MonoAOTCompiler.csproj : error MSB4057: The target "GetFilesToPackage" does not exist in the project` would throw.

Reply:
We could specify files to be packaged in the task project itself, like: https://github.com/radical/runtime/commit/b81753bbae2bd0f3c497e30054ae8af9e953daf5 .

Reply:
Thanks! I didn't realize that could be done 😄

Reply:
Added that commit and added a comment to explain the target.